addons/website/views/res_lang_views.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<odoo>
+    <!-- TODO: (review) in http_routing? -->
```

Comment:
yes, if field added in routing, take sense...

Reply:
Done